circuits/cpp/barretenberg/cpp/src/barretenberg/stdlib/primitives/uint/uint.test.cpp

```diff
@@ -1849,7 +1850,11 @@ TYPED_TEST(stdlib_uint, test_divide_special)
 }
 TYPED_TEST(stdlib_uint, div_remainder_constraint)
 {
-    TestFixture::div_remainder_constraint();
+    if constexpr (IsSimulator<CircuitSimulatorBN254>) {
```

Comment:
Probably the right thing to do is not compile tests we don't expect to pass, so that we can reserve "skip"s for tests we want to remind ourselves are being skipped for whatever reason.